record_with_rewards.sh

```diff
@@ -0,0 +1,12 @@
+python lerobot/scripts/control_robot.py record \
```

Comment:
Could you remove this file but add the command in the PRs description for testing?